### PR TITLE
Add gatekeeper docs

### PR DESCRIPTION
**Pull request by nsklikas**
Adds the gatekeeper docs.

- [ ] Should I add a link in the sidebar?

### CODE REVIEW

**Comment by evilnick** [comment]
Hmm. It is easy enough to add this to the navigation, but I'm not sure where to link it from exactly.  I.e, it isn't a core component or a cdk-addon

File: pages/k8s/gatekeeper.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+If using RBAC, you must deploy the charms using the `--trust` flag as the charm needs permissions
+in order to create the necessary resources:
+
+```console
+juju deploy --trust ch:gatekeeper-controller-manager
+juju deploy --trust ch:gatekeeper-audit
+```
+
+#### Storage Pool
+
```
Comment (evilnick) [inline pages/k8s/gatekeeper.md:60]: Is there something to go in here?

Comment (nsklikas) [reply pages/k8s/gatekeeper.md]: No, I removed it.

**Comment by evilnick** [comment]
Thanks for this, I will merge this page for now and we can add it to the security stuff when I fix that up :+1:

**Pull request merged**